### PR TITLE
Add about Ishaan

### PR DESCRIPTION
**Pull request by Ishaan0905**
I have added my info.

### CODE REVIEW

File: submissions/Ishaan0905/about.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# About Me
+My name is Ishaan Goyal. I am an ECE student at Panjab University, India.
+
+# Experience
+I am a Student at UIET(Best Experience)
```
Comment (divyam3897) [inline submissions/Ishaan0905/about.md]: full stop missing at the end of sentence.

File: submissions/Ishaan0905/about.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+# Interests
+Open source, Gaming, Food
```
Comment (divyam3897) [inline submissions/Ishaan0905/about.md]: Full stop

**Review by divyam3897 — CHANGES_REQUESTED** [review]

**Comment by divyam3897** [comment]
@Ishaan0905 as told commit message should be in present tense.

**Comment by Ishaan0905** [comment]
I have made the required changes

**Review by divyam3897 — CHANGES_REQUESTED** [review]
Remove empty lines and squash  your commits into one.

**Comment by Ishaan0905** [comment]
Please review.

**Pull request closed without merging**